source/administration/devops-command-center.rst

```diff
@@ -64,48 +64,128 @@ Viewing and creating playbooks
+   - To delete: Hover over an input box and select the **X** to remove the checklist item.
+   - To move: Drag the hamburger menu to rearrange the checklist items.
+3. Choose **Done** to save the changes.
 
 Ending incidents
 ~~~~~~~~~~~~~~~~
 
 There are two ways to end an incident:
 
-- Using slash command */incident end* from within the incident channel.
+- Using the slash command */incident end* from within the incident channel.
 - Using the **End Incident** button in the RHS panel while in the incident channel.
 
-The incident will become inactive, be removed from the list of active incidents, and the associated channel will be archived.
+The incident will become inactive, is removed from the list of active incidents, and the associated channel is archived.
 
 Administrator's Guide
 ^^^^^^^^^^^^^^^^^^^^^^
 
+Incidents and playbooks are associated with teams in Mattermost. Participants can be added to an incident by other incident members inviting them to the incident's channel. By default, all incidents are public (available to all members of a team). An incident can be made private during creation, or it can be made private after it has been created. 
+
 Permissions
 ~~~~~~~~~~~~~~~~~~~~~
 
-Incidents and playbooks are associated with teams in Mattermost. Participants can be added by inviting them to the incident's channel.
+System Admins and Team Admins can edit the visibility of, and access to, playbooks and incidents so that:
+
+- Users who are removed from an incident channel by a System Admin can no longer access the incident's playbook.
+- Team members who aren't System or Team Admins can't invite other team members to manage private playbooks.
+- Team members who aren't System or Team Admins can't be removed from an incident by other members of the team.
+- Team members who aren't System or Team Admins can't have access to playbooks removed by other members of the team.
+
+In addition, System and Team Admins can allow:
+
+- Public playbooks to be deleted by team members whether they're participants in an incident channel or not.
+
+Managing incident channel visibility
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-To view incidents associated with your Mattermost team, select **!** in the channel header to open the RHS menu. If there are no active incidents, you can start one or you can view details of incidents that have ended.
+System Admins can manage the visibility of channels by converting them into private incident channels. When a public incident is converted to a private incident, it can't be accessed by members who weren't in the channel at the time of the conversion. In addition, the contents of the channel won't show up in a search query or in the incident list in the incident backstage. 
 
-Incidents Backstage
+Once a public incident is converted to a private incident the change cannot be reversed. Non-participants who are in the same team can be added to the channel by an admin.
```

Comment:
Confirm that this is correct (not able to reverse the change).

Reply:
I don't really know. I'll look into this, but it's a general feature of Mattermost in any case.

Reply:
It's not possible to do this within the UX, but it can be done by a system administrator with access to the CLI. Again, this is technically all documented as part our existing channel permissions behaviour (nothing new), so propose we defer to that documentation?